### PR TITLE
High: jira: new resource agent

### PR DESCRIPTION
**Pull request by salehabbas**
This is a new ocf ra for managing JIRA software.

### CODE REVIEW

File: doc/man/Makefile.am [hunk 1]
```diff
@@ -120,6 +120,7 @@ man_MANS	       = ocf_heartbeat_AoEtarget.7 \
                           ocf_heartbeat_ids.7 \
                           ocf_heartbeat_iscsi.7 \
                           ocf_heartbeat_jboss.7 \
+			  ocf_heartbeat_jira.7 \
```
Comment (krig) [inline doc/man/Makefile.am]: Should be spaces, not tabs here

File: heartbeat/jira [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+
+    return $OCF_SUCCESS
+}
+
+# Meta-data
+jira_meta_data(){
+    cat <<EOF
+<?xml version="1.0"?>
+<!DOCTYPE resource-agent SYSTEM "ra-api-1.dtd">
+<resource-agent name="jira" version="0.1">
```
Comment (krig) [inline heartbeat/jira]: There shouldn't be a version attribute here. This is why the CI check fails, so if you remove the version here I think it should pass.

File: heartbeat/jira [hunk 3]
```diff
@@ -0,0 +1,266 @@
+#! /bin/bash
+# 
+####################################################################
+# Description: OCF Resource Agent to manage JIRA software.
+# Author     : Saleh A. (saleh.abbas.saber@gmail.com)
+#
+# License    : WTFPL 2
+#
+#        DO WHAT THE FUCK YOU WANT TO PUBLIC LICENSE
```
Comment (krig) [inline heartbeat/jira]: Nice choice of license, haha :)

File: doc/man/Makefile.am (hunk 1 above)
Comment (salehabbas) [reply doc/man/Makefile.am]: Fixed, thanks!

File: heartbeat/jira (hunk 3 above)
Comment (salehabbas) [reply heartbeat/jira]: :)

File: heartbeat/jira (hunk 2 above)
Comment (salehabbas) [reply heartbeat/jira]: Fixed, thanks!

File: heartbeat/jira [hunk 4]
```diff
@@ -0,0 +1,281 @@
...
+
+    # if resource is already running, bail out early
+    if jira_monitor; then
+        ocf_log info "Resource is already running"
+        return $OCF_SUCCESS
+    fi
+
+    # Starting Jira
+    waittime=300
+    /bin/su -m $jira_user -c "$jira_installation/bin/startup.sh &> /dev/null"
```
Comment (oalbrigt) [inline heartbeat/jira]: There's no reason to use full path for su, and it might cause issues for some users.

Comment (salehabbas) [reply heartbeat/jira]: Removed. Thanks

**Comment by oalbrigt** [comment]
LGTM.

**Comment by krig** [comment]
I just noticed that the agent uses `curl`, this should probably be noted in the description so that a user knows what the dependencies for using the agent are.

But since it's just a minor detail I'll merge this as it is now, the agent itself looks good to me too.

**Pull request merged**